### PR TITLE
Add demo app SSL experiment.

### PR DESCRIPTION
**Pull request by kierenevans**

### CODE REVIEW

File: nginx-ingress/demo-app-ssl/feature-add-ingress.ingress-if-port-80-only.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  annotations:
+    kubernetes.io/ingress.class: nginx
+    ingress.kubernetes.io/secure-backends: "false"
```
Comment (sroze) [inline nginx-ingress/demo-app-ssl/feature-add-ingress.ingress-if-port-80-only.yaml:6]: That's the default value, right?

File: nginx-ingress/demo-app-ssl/feature-add-ingress.sslcert.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+data:
+  tls.crt: [...generate with 'openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout key -out crt -subj "/CN=something.continuouspipe.net/O=nginxsvc"'...]
```
Comment (sroze) [inline nginx-ingress/demo-app-ssl/feature-add-ingress.sslcert.yml]: Where does this `nginxsvc` come from?

File: nginx-ingress/demo-app-ssl/feature-add-ingress.ingress-if-port-80-only.yaml (hunk 1 above)
Comment (kierenevans) [reply nginx-ingress/demo-app-ssl/feature-add-ingress.ingress-if-port-80-only.yaml:6]: Yes, just for documentation purposes.

File: nginx-ingress/demo-app-ssl/feature-add-ingress.sslcert.yml (hunk 2 above)
Comment (kierenevans) [reply nginx-ingress/demo-app-ssl/feature-add-ingress.sslcert.yml]: Need to provide an Organisation in the request, so it can be anything.

**Pull request closed without merging**